Package.resolved

```diff
@@ -41,8 +41,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/pointfreeco/xctest-dynamic-overlay",
       "state" : {
-        "revision" : "302891700c7fa3b92ebde9fe7b42933f8349f3c7",
-        "version" : "1.0.0"
+        "revision" : "23cbf2294e350076ea4dbd7d5d047c1e76b03631",
```

Comment:
@roanutil Did you want to include a package upgrade in this PR? Perfectly fine if so, I thought you had mentioned in the past doing this in a separate PR.

Reply:
@raj3m8 I usually don't mix. But this is a pretty benign dependency for a non-prod library.

If you'd rather a separate PR for that, I can do that.

Reply:
Yup, I agree. Can include in this PR.